### PR TITLE
Correcting polish translation - branch 4.4

### PR DESCRIPTION
**Pull request by qbadev**
General correction of polish translation. Mainly misspelings, but also punctation and factual corrections.

### CODE REVIEW

**Comment by leofeyer** [comment]
Thank you for your contribution. Unfortunately, translations are managed on transifex.com (see https://www.transifex.com/contao/) and cannot be changed in a pull request. If you sign up on transifex.com, I will happily add you to the Polish team so you can make the changes there.

**Comment by qbadev** [comment]
That's a shame. I hope three is a batch import of translations on transifex, so I don't have to input all items separately again.

**Comment by leofeyer** [comment]
@qbadev I doubt that there is. However, I might be able to checkout your PR and push the changes. Shall I try this?

@qzminski Are you ok with the changes?

**Comment by qbadev** [comment]
I have seen that there is export to CSV and then import from CSV on transifex, but only reviewers have access to import, I suppose.

**Comment by qbadev** [comment]
@leofeyer so I may export all data, translate it localy, then send to @qzminski

**Comment by leofeyer** [comment]
Ok. If that does not work, I can still try checking out your PR and pushing the changes. Are you sure that you do not want me to try it first? Would save you the work…

**Comment by qbadev** [comment]
@leofeyer I don't want to break any Contao development workflow, so I will first try to adapt to Your team. Let's first give it a try to export to CSV.

If You check out my PR, transifex will stil be out of date.

**Comment by qzminski** [comment]
First of all, big thanks to you for contributing to Polish translation. It's great that someone finally could spare some time on this one and provide some additional feedback as maintaining thousands of labels is pretty challenging and tedious work.

@leofeyer, if it's possible to merge this pull request, I would highly appreciate it as it would save @qbadev plenty of (potentially unnecessary) extra work. But before doing so, please add me as a reviewer as I have spotted a few unclear things there.

File: core-bundle/src/Resources/contao/languages/pl/default.xlf [hunk 1]
```diff
@@ -1203,15 +1203,15 @@
       </trans-unit>
       <trans-unit id="MSC.id.1">
         <source>Note that changing the ID can break data integrity!</source>
-        <target>ID jest unikatowym numerycznym identyfikatorem rekordu tabeli. Numery ID są zwykle przydzielane automatycznie. Jako, że numery ID używane są w zależnościach pomiędzy rekordami, zmiana lub ich usunięcie może doprowadzić do utraty integralności danych a co za tym idzie do awarii systemu!!</target>
+        <target>ID jest unikatowym numerycznym identyfikatorem rekordu tabeli. Numery ID są zwykle przydzielane automatycznie. Jako że numery ID używane są w zależnościach pomiędzy rekordami, zmiana lub ich usunięcie może doprowadzić do utraty integralności danych a co za tym idziedo awarii systemu!</target>
```
Comment (qzminski) [inline core-bundle/src/Resources/contao/languages/pl/default.xlf]: Missing space between `idzie` and `do`.

File: core-bundle/src/Resources/contao/languages/pl/default.xlf [hunk 2]
```diff
@@ -1533,7 +1533,7 @@ Ten e-mail został wygenerowane przez Contao. Nie odpowiadaj na niego.
       </trans-unit>
       <trans-unit id="MSC.save">
         <source>Save</source>
-        <target>Zapisz</target>
+        <target>Zachowaj</target>
```
Comment (qzminski) [inline core-bundle/src/Resources/contao/languages/pl/default.xlf]: Please revert back to `Zapisz` as it seems more common in various software in general. At least to me 😉 

Could you then please change `Zachowaj` to `Zapisz` in other places as well (e.g. `Zachowaj i nowy`)?

File: core-bundle/src/Resources/contao/languages/pl/exception.xlf [hunk 3]
```diff
@@ -35,11 +35,11 @@
       </trans-unit>
       <trans-unit id="XPT.requestToken">
         <source>Invalid request token</source>
-        <target>Nieprawidłowy request token</target>
+        <target>Nieprawidłowy token żądania</target>
       </trans-unit>
       <trans-unit id="XPT.invalidToken">
         <source>The request token could not be verified.</source>
-        <target>Request token nie został zweryfikowany.</target>
+        <target>Token żadania nie został zweryfikowany.</target>
```
Comment (qzminski) [inline core-bundle/src/Resources/contao/languages/pl/exception.xlf]: Typo in `żadania` –> `żądania`.

File: core-bundle/src/Resources/contao/languages/pl/tl_article.xlf [hunk 4]
```diff
@@ -111,31 +111,31 @@
       </trans-unit>
       <trans-unit id="tl_article.cssID.1">
         <source>Here you can set an ID and one or more classes.</source>
-        <target>Jeśli potrzebujesz osobnej definicji do formatowania tego modułu w stylach CSS, możesz tu wprowadzić ID. To ID będzie przypisane do tego modułu i można je modyfikować w arkuszu CSS.</target>
+        <target>Jeśli potrzebujesz osobnej definicji do formatowania tego modułu w stylach CSS, możesz tu wprowadzić ID. To ID będzie przypisane do tego modułu i można je opisywać w arkuszu CSS.</target>
```
Comment (qzminski) [inline core-bundle/src/Resources/contao/languages/pl/tl_article.xlf]: Maybe `używać` would be more appropriate after all.

File: core-bundle/src/Resources/contao/languages/pl/tl_image_size.xlf [hunk 5]
```diff
@@ -3,7 +3,7 @@
     <body>
       <trans-unit id="tl_image_size.name.0">
         <source>Name</source>
-        <target>Imię i nazwisko</target>
+        <target>Nazwa wyświetlana</target>
```
Comment (qzminski) [inline core-bundle/src/Resources/contao/languages/pl/tl_image_size.xlf]: Please change to just `Nazwa`.

File: core-bundle/src/Resources/contao/languages/pl/tl_style_sheet.xlf [hunk 6]
```diff
@@ -3,7 +3,7 @@
     <body>
       <trans-unit id="tl_style_sheet.name.0">
         <source>Name</source>
-        <target>Imię i nazwisko</target>
+        <target>Nazwa wyświetlana</target>
```
Comment (qzminski) [inline core-bundle/src/Resources/contao/languages/pl/tl_style_sheet.xlf]: Please change to just `Nazwa`.

File: core-bundle/src/Resources/contao/languages/pl/tl_style_sheet.xlf [hunk 7]
```diff
@@ -67,7 +67,7 @@
       </trans-unit>
       <trans-unit id="tl_style_sheet.title_legend">
         <source>Name</source>
-        <target>Imię i nazwisko</target>
+        <target>Nazwa wyświetlana</target>
```
Comment (qzminski) [inline core-bundle/src/Resources/contao/languages/pl/tl_style_sheet.xlf]: Please change to just `Nazwa`.

File: core-bundle/src/Resources/contao/languages/pl/tl_image_size.xlf (hunk 5 above)
Comment (qbadev) [reply core-bundle/src/Resources/contao/languages/pl/tl_image_size.xlf]: I found it in newest translation from Contao 4.10, that's why I put it in my correction.

File: core-bundle/src/Resources/contao/languages/pl/default.xlf (hunk 2 above)
Comment (qbadev) [reply core-bundle/src/Resources/contao/languages/pl/default.xlf]: `Save` is fine to me to be `Zapisz`, I will correct all of its occurences.

File: core-bundle/src/Resources/contao/languages/pl/tl_article.xlf (hunk 4 above)
Comment (qbadev) [reply core-bundle/src/Resources/contao/languages/pl/tl_article.xlf]: `używać` is fine

**Comment by qbadev** [comment]
I made all changes suggested by @qzminski. 
Now I will try to prepare CSV files for transifex.

**Comment by qbadev** [comment]
@leofeyer @qzminski according to [this](https://docs.transifex.com/translation/importing-translations) doc, reviewer or project owner can upload `XLIFF` files to **transfiex** to update translations.

**Comment by qzminski** [comment]
@qbadev can you push your changes to this repo please? I think @leofeyer would be able to merge them then.

**Comment by qbadev** [comment]
I already pushed them to my repo qbadev/contao:4.4, if this is what You ask for.

**Comment by leofeyer** [comment]
See #2242

**Pull request closed without merging**